catala/correr/correr.md

```diff
@@ -3,3 +3,7 @@ La ostia que et fots és de campionat!
 Que no saps que les parets no poden ser de núvol de sucre?
 
 És clar que no, si no les formigues t'entrarien a casa ;-P
+
+Què fas?
+
+[Et poses a buscar formigues](formigues/formigues.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.